front/cypress/integration/sign_in.ts

```diff
@@ -16,27 +16,24 @@ describe('Sign in page', () => {
   });
 
   it('shows an error when no email is provided', () => {
+    cy.get('#password').type('test');
     cy.get('#e2e-signin-password-submit-button').click();
 
     const errorMessage = cy.get('#email').siblings('.e2e-input-error');
-
     expect(errorMessage).to.exist;
   });
 
   it('shows an error when no valid email is provided', () => {
     cy.get('#email').type('test');
+    cy.get('#password').type('test');
     cy.get('#e2e-signin-password-submit-button').click();
 
-    const errorMessage = cy
-      .get('#password')
-      .parent()
-      .parent()
-      .siblings('.e2e-error-message');
-
+    const errorMessage = cy.get('#email').siblings('.e2e-input-error');
```

Comment:
You could call this `emailErrorMessage` (and the same with other errorMessages). That would perhaps have caught it if you had to explicitly think about what error message it is. :)

Reply:
Another good point! Changed it.